results/code-size.tex

```diff
@@ -155,6 +164,37 @@ \subsection{Executables}
 
 We see from \autoref{tab:size:c-vs-rust} that both the Circle Game and the Minimal Main  {\rust} binaries come quite close to the {\C} binaries in size.
 
+To see which protions of the {\rust} code that make the binary size increase, we now present a break down of the Sensor Tracker builds optimized for size.
```

Comment:
Litt rar setning ("Too see which..., we now present..."). Muligens bytte plass på delene, men vet ikke om det blir bedre.

også typos: protions -> portions og break down -> breakdown